lib/Fmt_ast.ml

```diff
@@ -537,6 +537,15 @@ let sequence_blank_line c (l1 : Location.t) (l2 : Location.t) =
       loop l1.loc_end (Cmts.remaining_before c.cmts l2)
   | `Compact -> false
 
+let fmt_quoted_string key ext s = function
+  | None | Some "" ->
```

Comment:
Is `Some ""` possible ?

Reply:
Looks like it's not possible indeed, I will simplify